ni/measurementlink/discovery/v1/discovery_service.proto

```diff
@@ -81,15 +84,21 @@ message ServiceDescriptor {
 
   // Optional. Represents a set of annotations on the service.
   // Well-known annotations:
+  //
   // - Description
+  //
   //   - Key: "ni/service.description"
   //   - Expected format: string
```

Comment:
FYI, I forgot reST nested lists need a blank line. I checked that VS Code's Markdown preview renders this correctly.